src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/RefVsAnyResult.java

```diff
@@ -0,0 +1,69 @@
+package org.broadinstitute.hellbender.tools.walkers.haplotypecaller;
+
+/**
+ * Holds information about a genotype call of a single sample reference vs. any non-ref event
+ */
+final class RefVsAnyResult {
+    /**
+     * The genotype likelihoods for ref/ref ref/non-ref non-ref/non-ref
+     */
+    private final double[] genotypeLikelihoods;
+
+    /**
+     * AD field value for ref / non-ref
+     */
+    private final int[] AD_Ref_Any = new int[2];
+
+
+    /**
+     * Creates a new ref-vs-alt result indicating the genotype likelihood vector capacity.
+     * @param likelihoodCapacity the required capacity of the likelihood array, should match the possible number of
+     *                           genotypes given the number of alleles (always 2), ploidy (arbitrary) less the genotyping
+     *                           model non-sense genotype count if applies.
+     * @throws IllegalArgumentException if {@code likelihoodCapacity} is negative.
+     */
+    public RefVsAnyResult(final int likelihoodCapacity) {
+        if (likelihoodCapacity < 0) {
+            throw new IllegalArgumentException("likelihood capacity is negative");
+        }
+        genotypeLikelihoods = new double[likelihoodCapacity];
+    }
+
+    /**
+     * @return Get the DP (sum of AD values)
+     */
+    int getDP() { return AD_Ref_Any[0] + AD_Ref_Any[1]; }
+
+    /**
+     * Return the AD fields. Returns the live array.
+     */
+    int[] get_AD_Ref_Any(){
+        return AD_Ref_Any;
+    }
+
```

Comment:
Change the method name to something more standard, the obvious choice is ```getAD```.

```
int[] getAD() {
   new int[] { refDepth, nonRefDepth };
}
```
Change the java doc accordingly; is not a "live" array any longer.